datetimepicker/src/main/java/com/taransit/datetimepicker/ViewPagerAdapter.kt

```diff
@@ -7,7 +7,7 @@ import androidx.recyclerview.widget.RecyclerView
 
 class ViewPagerAdapter : RecyclerView.Adapter<ViewPagerAdapter.PagerViewHolder>() {
 
-    val TABS = listOf("Departure", "Arrival")
+    val TABS = listOf("Departure at", "Arrival by")
```

Comment:
V této chvíli už by se hodilo strings.xml. Přinejmenším.

V ideálním případě, pokud by se ti do toho chtělo, jsem plánoval získávat info o tabech zvenčí při vytváření dialogu. Default je bez tabů a pokud nějaké volající přidá, zobrazí se a zároveň se pozice zvoleného tabu vybírá při vytvoření a vrací při kliknutí na OK. Ale tohle jen pokud by se ti chtělo. ;)